app/institutions/dashboard/-components/institutional-users-list/template.hbs

```diff
@@ -0,0 +1,68 @@
+<PowerSelect
+        @options={{this.departments}}
+        @searchEnabled={{false}}
+        @selected={{this.department}}
+        @triggerClass={{local-class 'select'}}
+        @onchange={{action 'onSelectChange'}} as |department|
```

Comment:
:soon: (after upgrades) we won't need the action helper at all. This prepares for that:
```suggestion
        @onchange={{action this.onSelectChange}} as |department|
```